src/main/kotlin/com/chattriggers/ctjs/api/commands/DynamicCommands.kt

```diff
@@ -841,13 +842,13 @@ object DynamicCommands : CommandCollection() {
     data class ItemStackArgumentWrapper(private val impl: ItemStackArgument) : Predicate<Item> {
         val itemType = ItemType(impl.item)
 
-        override fun test(item: Item) = impl.test(item.toMC())
+        override fun test(item: Item) = ItemStack.areItemsAndComponentsEqual(itemType.asItem().toMC(), item.toMC())
 
         fun test(type: ItemType) = itemType.getRegistryName() == type.getRegistryName()
     }
 
     data class MessageFormatArgumentWrapper(private val impl: MessageArgumentType.MessageFormat) {
-        val text: String by impl::contents
+        var text = impl.contents
```

Comment:
What's the reason for this change?

Reply:
We can't delegate the property since `MessageArgumentType.MessageFormat` is a Java Record

Reply:
Gotcha. Does it need to be a `var` though?

Reply:
Yes since you couldn't set the text otherwise. Honestly I don't know much about these argument wrappers, I just saw that it used delegation to be able to get and set, so I used a var to mimic that.

Reply:
Oh I didn't realize these classes were public API, nvm then